backend/Squirrel.Core/Squirrel.Core.BLL/Services/AzureStorageService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using AutoMapper;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.Common.DTO.Blob;
+using Squirrel.Core.DAL.Context;
+
+namespace Squirrel.Core.BLL.Services
+{
+    public class AzureStorageService : BaseService, IStorageService
+    {
+        private readonly BlobServiceClient _blobServiceClient;
+
+        public AzureStorageService(SquirrelCoreContext context, IMapper mapper, BlobServiceClient blobServiceClient) : base(context, mapper)
+        {
+            _blobServiceClient = blobServiceClient;
+        }
+
+        public async Task DeleteAsync(string container, string id)
+        {
+            var containerClient = await GetOrCreateContainerByNameAsync(container);
```

Comment:
`var containerClient = await GetOrCreateContainerByNameAsync(container);` looks like duplicates, can we move it into the constructor?

Reply:
We can`t move it into the constructor because we may have different containers in every call, so I`ll move this part into the private method.